client/rpc-servers/src/lib.rs

```diff
@@ -26,7 +26,7 @@ use log::error;
 use pubsub::PubSubMetadata;
 
 /// Maximal payload accepted by RPC servers.
-const MAX_PAYLOAD: usize = 15 * 1024 * 1024;
+const MAX_PAYLOAD: usize = 1024 * 1024;
```

Comment:
Why? Won't it fail to fit runtime upgrade?

Reply:
1) It's just an experiment to see, if it eliminates problems with memory on rpc nodes, 2) it's not the max length of the message, it's just the max length of the buffer for it, if the message is bigger, it will be just chunked.

Reply:
PS I'm not going to merge it for now))

Reply:
@grbIzl so this is good to go on the kusama RPC node as requested?